themes/fikaworks/layouts/members/list.html

```diff
@@ -19,7 +19,7 @@ <h1 class="mt-2 text-6xl font-extrabold tracking-tight text-gray-900">{{ .Title
               {{- if $src }}
                 {{- $small := $src.Fit "320x320" }}
                 <img
-                  class="w-100 h-100 rounded-full mx-auto {{ .Params.member_status }} membership"
+                  class="w-40 h-40 rounded-full mx-auto {{ .Params.member_status }} membership"
```

Comment:
Would it be better to use `object-cover`? our dear @mad-it  look a bit squeezed

see https://tailwindcss.com/docs/object-fit#resizing-to-cover-a-container

Reply:
Good point! Fixed